js/ui/grid_core/ui.grid_core.grid_view.js

```diff
@@ -64,8 +64,7 @@ const ResizingController = modules.ViewController.inherit({
             this._refreshSizesHandler = (e) => {
                 dataController.changed.remove(this._refreshSizesHandler);
 
-                const templateDeferreds = e && e.templateDeferreds || [];
-                when.apply(this, templateDeferreds).done(() => {
+                this.getView('rowsView')._waitAsyncTemplates(e || {}).done(() => {
```

Comment:
I think it's better to use a declared property: this._rowsView.
And make the _waitAsyncTemplates method public (remove prefix '_')